docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
       timeout: 5s
       retries: 3
       start_period: 3s
+    env_file:
```

Comment:
As mentioned in another comment, we could pass the values as environment variables. May be we don't set anything in docker compose and just update README saying that we could add ADMIN_USER and ADMIN_PASSWORD environment variables for custom values.

Reply:
Can't we use dotenv so we have both behaviour?

Reply:
`dotenv` package takes what is in the .env file and puts it in process.env. So, not sure if this will overwrite existing environment variables or the other way round. I feel like having two separate ways might cause a little confusion as to which value would be used.

Reply:
I did some reading on dotenv documentation and it turns out that by default it does not override the environment variables - https://www.npmjs.com/package/dotenv#override
 
Also, I played with environment variables a bit. I added environment variables for admin user and password in Windows PowerShell. I also added .env file in KORT repo and added the variables there with a different value. When running node application from the same PowerShell window, the local values are used rather than the one we provided in the file. 
 
So Remi is right that the environment variables will have precedence.

Reply:
I also prefer `dotenv`, but I'm happy to revert to environment variable ( similar to `mongoHost` ) and remove the references to `.env `

Reply:
Yeah, lets keep the dotenv. Even if we provide an environment variable we do not need to change the code for that, it should automatically work.